Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -561,7 +561,10 @@ public void InitializeColorScheme()
         private void QueryTextBox_KeyUp(object sender, KeyEventArgs e)
         {
             if(_viewModel.QueryText != QueryTextBox.Text)
-                _viewModel.QueryText = QueryTextBox.Text;
+            {
+                BindingExpression be = QueryTextBox.GetBindingExpression(System.Windows.Controls.TextBox.TextProperty);
+                be.UpdateSource();
+            }
```

Comment:
Sorry I mean these two

Reply:
I just tested it and it works the same :'( There is no difference.